docs/stories/EPIC-Label/TF-1641-user-can-view-all-messages-of-one-label.md

```diff
@@ -0,0 +1,42 @@
+# Summary
+
+* [Related EPIC](#related-epic)
+* [Definition](#definition)
+* [Screenshots](#screenshots)
+* [Misc](#misc)
+
+## Related EPIC
+
+https://github.com/linagora/tmail-flutter/issues/1638
+
+## Definition
+
+- Given that I am a Tmail user and I have logged in successfully
+- I  click on one label in left panel, then the screen that listing all emails which have this label will be shown
+- Or in the content view of an email, when I hover the label, there wull be a tooltip saying :"Go to label"
+- I click on the label, then the screen listing all emails which have this label is displayed.
+- On this email list, I can see the name of original mailbox of each email on the list 
+
+
+[Back to Summary](#summary)
+
+## Screenshots
+
+![image](https://user-images.githubusercontent.com/68209176/228921162-5033893b-60fa-4ee0-bc8f-d0503b46617f.png)
+
+
+![image](https://user-images.githubusercontent.com/68209176/228170226-dafddf6c-21bd-4ff5-a2fc-055d93de8fb0.png)
+
+### Mobile 
+
+![image](https://user-images.githubusercontent.com/68209176/229013124-2ffaf085-e088-4770-9181-81818a93678a.png)
```

Comment:
I think the title of the page should be changed to name of label. WDYT?

Reply:
story is updated @hoangdat